server/testdriver/junit4/src/main/java/org/infinispan/server/test/junit4/InfinispanServerRule.java

```diff
@@ -78,8 +78,8 @@ public void evaluate() throws Throwable {
             } finally {
                InfinispanServerRule.this.after(testName);
                if (manageServer && testServer.isDriverInitialized()) {
-                  testServer.afterListeners();
                   testServer.getDriver().stop(testName);
+                  testServer.afterListeners();
```

Comment:
Why is this change required?

Reply:
@ryanemerson Without this change we will receive `SocketTimeoutException: connect timed out` a lot